packages/flutter/test/services/message_codecs_test.dart

```diff
@@ -54,11 +54,14 @@ void main() {
         details: 'errorDetails',
       );
       expect(
-          () => method.decodeEnvelope(errorData),
-          throwsA(predicate((PlatformException e) =>
+        () => method.decodeEnvelope(errorData),
+        throwsA(predicate(
+          (PlatformException e) =>
```

Comment:
nit (unrelated to this PR, though): according to our style guide, the `=>` short hand shouldn't be used here because it doesn't fir on one line.

(and elsewhere in this file as well)

Reply:
I'll take a stab at a lint for that.